examples/with-slate/CustomKeygenEditor/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Plain from 'slate-plain-serializer'
+import { KeyUtils } from 'slate'
+import { Editor } from 'slate-react'
+
+class CustomKeygenEditor extends React.Component {
+  constructor (props) {
+    super(props)
+    let key = 0
+    const keygen = () => {
+      key += 1
+      return props.uniqueId + key // custom keys
```

Comment:
Why not use the React `key` property? it's unique already (`props.key`)

Reply:
`props.key` is not available

> `key` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)

Reply:
Ah ok 👍